src/main/java/lotto/domain/BonusNumber.java

```diff
@@ -0,0 +1,21 @@
+package lotto.domain;
+
+import java.util.List;
+import lotto.util.validator.LottoNumberValidator;
+
+public class BonusNumber {
+    private final int number;
+
+    public BonusNumber(int number) {
+        this.number = number;
+    }
+
+    public static BonusNumber from(int bonusNumber, List<Integer> numbers) {
+        LottoNumberValidator.validateBonusNumber(bonusNumber, numbers);
+        return new BonusNumber(bonusNumber);
+    }
+
+    public int getNumber() {
+        return number;
+    }
```

Comment:
당첨번호를 인자로 받아서 해당 리스트에 
BonusNumber 클래스의 number값이 있는지 확인하는 메세지를 보내도록해 
getter를 제거하는건 어떨까요!?

```suggestion
    public boolean isMatches(List<Integer> numbers) {
        return numbers.contains(this.number);
    }
```

Reply:
앗 다시 생각해보니 BonusNumber는 단순히 dto성에 가깝게 쓰고 있었는데 dto로 선언을 안해주다보니 이런 일이 발생하는군요! 감사합니다ㅎㅎ